test/alternator/test_metrics.py

```diff
@@ -129,6 +129,24 @@ def test_batch_write_item(test_table_s, metrics):
     with check_increases_operation(metrics, ['BatchWriteItem']):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}]})
+@contextmanager
+def check_histogram_value(metrics, metric_name, labels, expected_value):
+    the_metrics = get_metrics(metrics)
+    saved_metrics = get_metric(metrics, metric_name, labels, the_metrics)
+    yield
+    the_metrics = get_metrics(metrics)
+    assert expected_value == get_metric(metrics, metric_name, labels, the_metrics) - saved_metrics
```

Comment:
What in is this function is specific to "histogram" as its name suggests?

Seems to me this function is exactly check_increases_metric_exact() - which you recently added too - except making "labels" (which is always "None" in the existing function) a parameter.
Or am I missing something?

Reply:
you're right, I didn't want to abuse the existing function, I'll rename and make one function uses the other

Reply:
This is "your" function, so you can change it to a good way that you think will fit all "your" tests - you're not "abusing" someone else function here ;-)

But seriously,
1. I don't think this function has anything to do with "histograms" so its name shouldn't suggest this.
2. I think adding a "labels=None" parameter to an existing function that currently just uses "None" as a constant is simple enough, no need to add a second function that uses the first function - especially not if the second function has "histograms" in its name, which as I noted, really has nothing to do with what this function does (it's not specific to hisograms).

Reply:
That's what I ended up doing